content/en/user-guide/aws/cloudfront/index.md

```diff
@@ -70,14 +70,17 @@ This feature is still under development, and functionality is limited.
 You can enable this feature by setting `CLOUDFRONT_LAMBDA_EDGE=1` in your LocalStack configuration.
 
 ### Current features
+
 - Support for [`CreateDistribution`](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_CreateDistribution.html) API to set up CloudFront distributions with Lambda@Edge.
 - Support for modifying request and response headers dynamically.
+- Support for [`IncludeBody`](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_LambdaFunctionAssociation.html#cloudfront-Type-LambdaFunctionAssociation-IncludeBody) parameter.
+- Support for Node.js & Python 3.x runtime.
+
+### Current limitations
 
-### Known limitations
 - The [`UpdateDistribution`](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_UpdateDistribution.html), [`DeleteDistribution`](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_DeleteDistribution.html), and [`Persistence Restore`](https://docs.localstack.cloud/user-guide/state-management/persistence/) features are not yet supported for Lambda@Edge.
 - Lack of full support for `viewer-request` and `viewer-response` event types.
```

Comment:
@anisaoshafi Shouldn't these refer to the `origin-*` event types?

What do you think about something along these lines?
> The `origin-request` and `origin-response` event types currently trigger for each request because caching is not implemented in CloudFront.

That's more specific than "Please expect inconsistencies and missing functionality."

Reply:
@joe4dev correct me if I'm wrong, I think we had a different understanding of the events and we didn't yet know about the cache limitation at that time.
You're right, current statement is not correct and your suggestion is more concise:
`The origin-request and origin-response event types currently trigger for each request because caching is not implemented in CloudFront.`

I'll open a PR with this change.

Reply:
https://github.com/localstack/docs/pull/1757 🙏🏼